src/osversion/test/VersionTranslator_GTest.cpp

```diff
@@ -4022,3 +4022,139 @@ TEST_F(OSVersionFixture, update_3_6_1_to_3_7_0_HeatPumpPlantLoopEIR) {
     EXPECT_TRUE(hp.isEmpty(insertionIndex++));                           // TimedEmpiricalDefrostHeatInputEnergyFractionCurveName
   }
 }
+
+TEST_F(OSVersionFixture, update_3_7_0_to_3_8_0_NoLoadSupplyAirFlowRateControlSetToLowSpeed) {
+  openstudio::path path = resourcesPath() / toPath("osversion/3_8_0/test_vt_NoLoadSupplyAirFlowRateControlSetToLowSpeed.osm");
+  osversion::VersionTranslator vt;
+  boost::optional<model::Model> model = vt.loadModel(path);
+  ASSERT_TRUE(model) << "Failed to load " << path;
+
+  openstudio::path outPath = resourcesPath() / toPath("osversion/3_8_0/test_vt_NoLoadSupplyAirFlowRateControlSetToLowSpeed_updated.osm");
+  model->save(outPath, true);
+
+  std::vector<WorkspaceObject> ptacs = model->getObjectsByType("OS:ZoneHVAC:PackagedTerminalAirConditioner");
+  ASSERT_EQ(1u, ptacs.size());
+  WorkspaceObject ptac = ptacs[0];
+
+  EXPECT_EQ("Zone HVAC Packaged Terminal Air Conditioner 1", ptac.getString(1).get());  // Name
+  EXPECT_EQ("Always On Discrete", ptac.getTarget(2)->nameString());                     // Availability Schedule Name
+  EXPECT_TRUE(ptac.isEmpty(3));                                                         // Air Inlet Node Name
+  EXPECT_TRUE(ptac.isEmpty(4));                                                         // Air Outlet Node Name
+  EXPECT_EQ("OutdoorAir:Mixer", ptac.getString(5).get());                               // Outdoor Air Mixer Object Type
+  EXPECT_TRUE(ptac.isEmpty(6));                                                         // Outdoor Air Mixer Name
+  EXPECT_EQ("Autosize", ptac.getString(7).get());                                       // Supply Air Flow Rate During Cooling Operation
+  EXPECT_EQ("Autosize", ptac.getString(8).get());                                       // Supply Air Flow Rate During Heating Operation
+  EXPECT_EQ("Autosize", ptac.getString(9).get());                                       // Supply Air Flow Rate When No Cooling or Heating is Needed
+  EXPECT_EQ("Yes", ptac.getString(10).get());                                           // No Load Supply Air Flow Rate Control Set To Low Speed
```

Comment:
This is good and all, but you should test just the field before insertion, insertion, after insertion and last field. Otherwise this will create more annoying updates if later on another field is added.